consensus/view_change.go

```diff
@@ -341,16 +340,17 @@ func (consensus *Consensus) onViewChange(msg *msg_pb.Message) {
 	utils.Logger().Debug().
 		Str("validatorPubKey", senderKey.SerializeToHexStr()).
 		Msg("[onViewChange] Add M3 (ViewID) type message")
```

Comment:
do you need to add viewID into the log messages?  that can help us triage the issues later.

Reply:
yes, I added it.